application/forms/Authentication/LoginForm.php

```diff
@@ -27,6 +27,12 @@ class LoginForm extends Form
      */
     const REDIRECT_URL = 'dashboard';
 
+    public static $defaultElementDecorators = [
+        ['ViewHelper', ['separator' => '']],
+        ['Help', []],
+        ['Errors', ['separator' => '']],
+    ];
+
```

Comment:
You've removed the control-group, but why is this still necessary?

Reply:
Makes sense. Removing this also solves #4395.

Reply:
✅